tests/test_scenarios.py

```diff
@@ -37,7 +35,7 @@ def test_scenario(filename, monkeypatch):
     )
 
     if os.getenv("UPDATE_SCENARIOS"):
-        save_scenario(path, cleaned_output)
+        save_scenario(path, cleaned_output) # pragma: no cover
```

Comment:
more of a question, this no-covers just the line, whereas putting it on the `if` above also no-covers that missed branch?

Reply:
If you put the pragma on the `if`, it's saying the if will not run.  But the if does run, and we want to know if for some reason in the future it doesn't run.  The line we don't need to run is the `save_scenario`.  Coverage knows enough to not mark the `if` as a partial branch since one of the two destinations has been excluded.

Putting the pragma on the `if` would also exclude the save_scenario line, but excludes more than we want to.